src/Editor/EditorUtils.js

```diff
@@ -18,7 +18,7 @@ export const displayTextWithMentions = (inputText, formatMentionNode) => {
                 const initialStr = retLine.substring(lastIndex, men.start);
                 lastIndex = (men.end+1);                    
                 formattedText.push(initialStr);
-                const formattedMention= formatMentionNode(`@${men.username}`, `${index}-${men.id}-${rowIndex}`);
+                const formattedMention= formatMentionNode(`@${men.username}`, `${index}-${men.userId}-${rowIndex}`);
```

Comment:
I think we should not change this. As `id` is more generic than `userId`. It depends what kind of data structure someone has. 

However we can take `idKey` as a prop to use it to populate mention id. Pull request is welcome.

Reply:
Hey sorry, I guess you are right. Just change the findMentions function. This will now return id instead of `userId`. 
Pardon me for the earlier message.